example.ulc.lua

```
@@ -57,5 +59,14 @@ return {names = {""},
-  }
+  },
+  fakeEnvConfig = {
+    useFakeEnv = true, -- opt-in per vehicle, off by default
+    lights = {
+      { extra = 10, direction = 'left' },  -- left alley
+      { extra = 11, direction = 'tk' },    -- take-down
+      { extra = 12, direction = 'right' }, -- right alley
+    }
+  },
+}
 }
```

Luacheck (1.2.0)

[error] expected <eof> near '}'

(E011)